src/Orleans.Streaming/QueueBalancer/LeaseBasedQueueBalancerOptions.cs

```diff
@@ -34,14 +34,26 @@ public class LeaseBasedQueueBalancerOptions
         public static readonly TimeSpan DefaultLeaseRenewPeriod = TimeSpan.FromSeconds(29); 
 
         /// <summary>
-        /// Gets or sets how often balancer attempts to aquire leases.
+        /// Gets or sets how often balancer attempts to acquire leases.
         /// </summary>
-        public TimeSpan LeaseAquisitionPeriod { get; set; } = DefaultMinLeaseAquisitionPeriod;
+        public TimeSpan LeaseAcquisitionPeriod { get; set; } = DefaultMinLeaseAcquisitionPeriod;
```

Comment:
We will have to document this small breaking change in the release note

Reply:
The original properties are still there, just marked [Obsolete] with a note to use the correctly spelled ones.